Sources/SwiftProtobufCore/BinaryEncodingSizeVisitor.swift

```diff
@@ -98,7 +97,8 @@ internal struct BinaryEncodingSizeVisitor: Visitor {
     serializedSize += tagSize + Varint.encodedSize(of: Int64(count)) + count
   }
 
-  mutating func visitSingularBytesField(value: Data, fieldNumber: Int) throws {
+  @inlinable
+  mutating func visitSingularBytesField<Bytes: SwiftProtobufContiguousBytes>(value: Bytes, fieldNumber: Int) throws {
```

Comment:
Now that I think about it, if we're keeping bytes fields as `Data`, what's the value in making this generic like this?  Does it come with any costs?  (generated code size?  performance?)

Reply:
I think this is a fair concern. If we keep bytes fields as `Data` we should already have bridged to `Data` before calling this method.

Reply:
You're right, this was unnecessary - visitors will operate on the protos' fields, and bytes fields will remain as `Data`, so no need to change the `Visitor` protocol. I'll roll this back.

Reply:
You're right, this was unnecessary - visitors will operate on the protos' fields, and bytes fields will remain as `Data`, so no need to change the `Visitor` protocol. I'll roll this back.